services/QuillLMS/config/routes.rb

```diff
@@ -736,6 +736,13 @@
     end
   end
 
+  resources :admin_report_filter_selections, only: [] do
+    collection do
+      post '/show', to: 'admin_report_filter_selections#show'
```

Comment:
This almost certainly doesn't matter, but it feels like maybe this should be a `GET` call instead of `POST` since it doesn't change data and doesn't expect a large set of param data.  But keeping it POST for consistency would be fine.  If you do decide to change this, I think we can also remove this line and just add `:show` to the currently-empty `only: []` array since `show` is a "standard" action in Rails.